### PR TITLE
Adrian/hooks

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by carledriss** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [DeleteProjectTest.java#L4](https://github.com/AWT-01/pivotal-selenium-tests/blob/5c1bb9e2bb79857c52fc3083ff76d9a579552327/src/test/java/org/fundacionjala/pivotal/testng/DeleteProjectTest.java#L4): Remove this unused import 'org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by carledriss** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [DeleteProjectTest.java#L4](https://github.com/AWT-01/pivotal-selenium-tests/blob/172b668ece43a318277ba24151275267ce830526/src/test/java/org/fundacionjala/pivotal/testng/DeleteProjectTest.java#L4): Remove this unused import 'org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+
+public class CommonSteps {
+
+    private Dashboard dashboard;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java]: MINOR Remove the "dashboard" field and declare it as a local variable in the relevant methods. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)

File: src/test/java/org/fundacionjala/pivotal/testng/createWorkspaceTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+import org.fundacionjala.pivotal.pageobjects.workspaces.Workspaces;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class createWorkspaceTest {
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/testng/createWorkspaceTest.java]: MINOR Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00101)

**Comment by carledriss** [comment]
SonarQube analysis reported 8 issues
* ![MINOR][MINOR] 8 minor

Watch the comments in this conversation to review them.

#### 6 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [NewProjectForm.java#L6](https://github.com/AWT-01/pivotal-selenium-tests/blob/0c208f008bf7a0f8e92dc0a17f8b4696b5163e59/src/main/java/org/fundacionjala/pivotal/pageobjects/dashboard/NewProjectForm.java#L6): Remove this unused import 'org.fundacionjala.pivotal.pageobjects.workspaces.Workspaces'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [StoriesSteps.java#L4](https://github.com/AWT-01/pivotal-selenium-tests/blob/0c208f008bf7a0f8e92dc0a17f8b4696b5163e59/src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java#L4): Remove this unused import 'cucumber.api.java.en.Given'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [StoriesSteps.java#L6](https://github.com/AWT-01/pivotal-selenium-tests/blob/0c208f008bf7a0f8e92dc0a17f8b4696b5163e59/src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java#L6): Remove this unused import 'org.fundacionjala.core.ui.WebDriverManager'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [StoriesSteps.java#L7](https://github.com/AWT-01/pivotal-selenium-tests/blob/0c208f008bf7a0f8e92dc0a17f8b4696b5163e59/src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java#L7): Remove this unused import 'org.fundacionjala.core.util.Environment'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [StoriesSteps.java#L8](https://github.com/AWT-01/pivotal-selenium-tests/blob/0c208f008bf7a0f8e92dc0a17f8b4696b5163e59/src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java#L8): Remove this unused import 'org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [StoriesSteps.java#L9](https://github.com/AWT-01/pivotal-selenium-tests/blob/0c208f008bf7a0f8e92dc0a17f8b4696b5163e59/src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java#L9): Remove this unused import 'org.fundacionjala.pivotal.pageobjects.login.SignInPage'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by 11TK11 — APPROVED** [review]

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package org.fundacionjala.pivotal.cucumber.steps;
+
+import cucumber.api.java.en.Given;
+import org.fundacionjala.core.util.Environment;
+import org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java]: MINOR Remove this unused import 'org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by carledriss** [comment]
SonarQube analysis reported 2 issues
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import org.fundacionjala.pivotal.pageobjects.dashboard.NewProjectForm;
+import org.fundacionjala.pivotal.pageobjects.login.SignInPage;
+
+/**
+ * Common steps.
+ */
+public class CommonSteps {
+    private Dashboard dashboard;
+    private NewProjectForm formPage;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/CommonSteps.java]: MAJOR Remove this unused "formPage" private field. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)
MINOR Remove the "formPage" field and declare it as a local variable in the relevant methods. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java [hunk 5]
```diff
@@ -1,66 +1,42 @@
 package org.fundacionjala.pivotal.cucumber.steps;
 
 import cucumber.api.java.en.And;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/StoriesSteps.java]: MINOR Remove this unused import 'cucumber.api.java.en.And'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by carledriss** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by 11TK11 — APPROVED** [review]

**Pull request merged**